be/src/exec/schema_scanner/sys_fe_memory_usage.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/schema_scanner/sys_fe_memory_usage.h"
+
+#include "exec/schema_scanner/schema_helper.h"
+#include "gen_cpp/FrontendService_types.h"
+#include "runtime/runtime_state.h"
+#include "runtime/string_value.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SysFeMemoryUsage::_s_columns[] = {
+        {"module_name", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"class_name", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"current_consumption", TYPE_VARCHAR, sizeof(StringValue), true},
+        {"peak_consumption", TYPE_VARCHAR, sizeof(StringValue), true}};
```

Comment:
It is better to use int64 as consumption, then we can use SQL expression to operate on this integer.
For better print, we can provide another function to make it human-friendly.